react/src/lib/Giscus.tsx

```diff
@@ -22,29 +22,30 @@ export default function Giscus({
 
   useEffect(() => {
     if (mounted) return;
-    import('giscus');
-    setMounted(true);
+    import('giscus').then(() => setMounted(true));
   }, []);
 
-  if (!mounted) return null;
-
   return (
-    <giscus-widget
-      id={id}
```

Comment:
Thanks, but this wraps the component in a fragment unnecessarily. I'd much prefer the old explicit `return null`.

Reply:
OK, I revert to old explicit `return null` in https://github.com/giscus/giscus-component/pull/1987/commits/7286aebe8f2c862bf9d94e16efbb61097c801d1d.